sites/fast-website/src/app/components/navigation/navigation.ts

```diff
@@ -0,0 +1,6 @@
+import { FASTElement, observable } from "@microsoft/fast-element";
+
+export class Navigation extends FASTElement {
+    @observable
+    items: HTMLElement[];
```

Comment:
@khamu are we doing anything with `items` here? If we're just dropping anchors in then you can likely drop the slotted directive from the template and drop this here.

Reply:
not doing anything with the items, so will remove the slotted directive